### PR TITLE
Remove delivery-cli examples and tests

### PR DESCRIPTION
**Pull request by afiune**
Bump mixlib-install version to 1.1 since that version removes the delivery-cli from the product matrix: https://github.com/chef/mixlib-install/pull/117

### CODE REVIEW

**Comment by afiune** [comment]
@chef-cookbooks/engineering-services @chef-cookbooks/delivery 👀

File: Gemfile [hunk 1]
```diff
@@ -13,13 +13,10 @@ end
...
   gem 'mixlib-versioning'
-  gem 'mixlib-install', '~> 1.0'
+  gem 'mixlib-install', '~> 1.1'
   gem 'chef-sugar'
   gem 'chefspec', github: 'sersut/chefspec', branch: 'sersut/export-repo-compat'
   gem 'chef-dk'
-
-  # Pin nokogiri to 1.6.7.2 until the new version can install in travis.
-  gem 'nokogiri', '= 1.6.7.2'
 end
```
Comment (tylercloke) [inline Gemfile]: Can we ditch this pin altogether now?

Comment (afiune) [inline Gemfile]: just did! 👍

**Comment by tylercloke** [comment]
🚀

File: Gemfile (hunk 1 above)
Comment (tylercloke) [inline Gemfile]: 💯

**Comment by sersut** [comment]
👍

**Pull request merged**